tests/meta-mender-ci/recipes-testing/read-only-rootfs/mender-client_%.bbappend

```diff
@@ -3,7 +3,5 @@ FILES_${PN}_append_mender-testing-enabled = " /data/etc/mender "
 do_install_append_mender-testing-enabled() {
     mkdir -p ${D}/data/etc/mender
     mv ${D}/etc/mender/mender.conf ${D}/data/etc/mender
-    mv ${D}${MENDER_CERT_LOCATION} ${D}/data/etc/mender/server.crt
     ln -s /data/etc/mender/mender.conf ${D}/etc/mender/mender.conf
-    ln -s /data/etc/mender/server.crt ${D}${MENDER_CERT_LOCATION}
```

Comment:
Something tells me this will need a new symlink and an adjustment elsewhere. Usually these links exist so that we can modify the file on the data partition, since the rootfs is read-only during the acceptance tests.

I have not searched for it though, just a hunch.

Reply:
I also had the same hunch. But doing the same search than you and, most importantly, running all tests should be enough...

(two test failed but not related to this)